elasticdl/requirements.txt

```diff
@@ -6,3 +6,4 @@ Cython
 odps
 tensorflow==2.1.0
 psutil
+cloudpickle
```

Comment:
What is this package used for?

Reply:
Horovod needs the package.
https://github.com/horovod/horovod/blob/2bd305a4b5b247b1d78514f6e49e0969208d028c/horovod/runner/common/util/network.py#L22